cypress/e2e/registration_validation.spec.ts

```diff
@@ -137,7 +137,7 @@ describe('User opens registration form and can see validation errors', () => {
     );
 
     // NPI Subject
-    cy.selectNpiDiscipline('Lingvistikk');
+    cy.selectNpiDiscipline(document.documentElement.lang === 'en' ? 'Linguistics' : 'Lingvistikk');
```

Comment:
Sånn generelt burde vi ha minst mulig logikk for sånt i testene, så jeg tenker enten finne en faktisk id eller bare endre til den engelske om det er den nye default funksjonaliteten. 
```suggestion
    cy.selectNpiDiscipline('Linguistics');
```